prettify_components.sh

```diff
@@ -0,0 +1,18 @@
+# Add this as a shell script to your WebStorm configurations
+# and as executable command to VS Code
+# Make sure the working directory for this command is equal to "<PROJECT_DIR>/src/components"
+
+# The path to the prettier config file.
+CONFIG_FILE="../../.pretierrc"
+
+# The path to the prettier ignore file.
+IGNORE_FILE="../../.prettierignore"
+
+# The regex for the exact files to be handled by prettifier.
+# ** = all (sub-)directories in the current working directory.
+# *.tsx = all files which are appended with ".tsx" in the specified directory.
+COMPONENTS_DIR_REGEX="./**/*.tsx"
+
+prettier --config $CONFIG_FILE --ignore-path $IGNORE_FILE all --write $COMPONENTS_DIR_REGEX
```

Comment:
I'm a bit worried that not escaping it as `"$COMPONENTS_DIR_REGEX"` might lead to "glob expansion".

See [Shellcheck](https://www.shellcheck.net):

<img width="987" alt="Screenshot 2021-04-24 at 14 43 02" src="https://user-images.githubusercontent.com/30694/115959218-6f921500-a50b-11eb-80d0-2c53dfce0388.png">